indexer/variables.tf

```diff
@@ -572,3 +572,8 @@ variable "lambda_cpu_architecture" {
   }
 }
 
+variable "create_backup_full_node" {
+  description = "Create backup full node. Default: true for mainnet, testnet"
+  type        = bool
+  default     = true
```

Comment:
nit: I think the default should be false. Testnet and Mainnet get churned rarely

Reply:
Makes sense. Fixed in https://github.com/dydxprotocol/v4-infrastructure/pull/127